app/controllers/middleware_server_group_controller.rb

```diff
@@ -3,12 +3,44 @@ class MiddlewareServerGroupController < ApplicationController
   include Mixins::GenericShowMixin
   include EmsCommon
   include MiddlewareCommonMixin
+  include Mixins::MiddlewareDeploymentsMixin
 
   before_action :check_privileges
   before_action :get_session_data
   after_action :cleanup_action
   after_action :set_session_data
 
+  OPERATIONS = {
+    :middleware_server_group_reload         => {:op  => :reload_middleware_server_group,
+                                                :msg => N_('Reload')
```

Comment:
I am not the author of the concept, but afaik these are actually the strings that gets plugged into sentences. Here is an example where it's being used https://git.io/vMbLs